### PR TITLE
Culture page

### PR DESCRIPTION
**Pull request by jpan1997**
TODO items:
- Update the pillars section to reference sanity text instead of hardcoding the content
- Componentize venn diagram to be shared across culture page and home page
- Don't hardcode "we believe that culture is ...". Add field in sanity schema instead
- Add page footer banners

### CODE REVIEW

File: src/pages/about-us/culture.tsx [hunk 1]
```diff
@@ -25,14 +84,119 @@ export default function CulturePage({
   if (!sanityCulturePage)
     throw `No Sanity document for the culture page was found.`
 
+  const icons = [
+    <ChatBubble color="primary" fontSize="large" />,
+    // @ts-ignore Color tertiary added in module augmentation but doesn't show up here.
```
Comment (sloh4137) [inline src/pages/about-us/culture.tsx:89]: Oh oops it should because we added it

File: src/components/Card/AlternatingContentGrid.tsx [hunk 2]
```diff
@@ -39,10 +39,10 @@ export default function AlternatingContentGrid({
             alignItems="stretch"
             spacing={1}
             justifyContent="space-between"
-            // justifyContent={index % 2 === 0 ? "flex-start" : "flex-end"}
+          // justifyContent={index % 2 === 0 ? "flex-start" : "flex-end"}
```
Comment (sloh4137) [inline src/components/Card/AlternatingContentGrid.tsx:42]: what the fuck?? this is a level of sloppiness that we can no longer accept in our organization. dare to do this one more time and consider it your last..... time ever interacting with LYF again. how hard is it to double check your PRs like god damn justin ur a fukin mess

Comment (sloh4137) [reply src/components/Card/AlternatingContentGrid.tsx:42]: \- albert

File: src/pages/about-us/culture.tsx [hunk 3]
```diff
@@ -25,14 +84,119 @@ export default function CulturePage({
...
+        </Stack>
+      </Section>
+
+      {/* Blurb */}
+      <Section maxWidth="sm">
+        <Stack
+          spacing={3}
+        >
+          <Typography variant="h4" textAlign="center">{sanityCulturePage?.blurb?.title}</Typography>
+          <PortableText color="gray" textAlign="center" content={sanityCulturePage?.blurb?._rawBody} />
```
Comment (sloh4137) [inline src/pages/about-us/culture.tsx:113]: By the way if you want it to be forced to be a certain size you can add a minHeaderSize

File: src/pages/about-us/culture.tsx [hunk 4]
```diff
@@ -25,14 +84,119 @@ export default function CulturePage({
...
+                )}
+              </Stack>
+            </Stack>
+          </Grid>
+        </Grid>
+      </Section>
+
+
+      {/* Culture is */}
+      {/* backgroundColor is hardcoded to "offwhite" with 0.4 opacity */}
```
Comment (sloh4137) [inline src/pages/about-us/culture.tsx:174]: Heh nice

**Review by sloh4137 — APPROVED** [review]

**Review by sloh4137 — APPROVED** [review]

File: src/components/Card/AlternatingContentGrid.tsx (hunk 2 above)
Comment (jpan1997) [reply src/components/Card/AlternatingContentGrid.tsx:42]: I quit

**Pull request merged**